python/ccxt/async_support/base/exchange.py

```diff
@@ -2,7 +2,7 @@
 
 # -----------------------------------------------------------------------------
 
-__version__ = '1.17.419'
+__version__ = '1.17.420'
```

Comment:
This edit isn't really needed, as the version is single-sourced by the `vss.js` script upon build on Travis )

Reply:
Oh neato, thanks for letting me know.